clearwater-snmpd/usr/share/clearwater/infrastructure/scripts/snmpd_fd_handler

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Copyright (C) 2018  Metaswitch Networks Ltd
+# Handles SNMP requests offloaded by snmpd for the file descriptor OIDs
+oid_prefix=.1.2.826.0.1.1578918.17.1
+oid_req=$2
+
+if [ "$1" = "-n" ]; then
+  # If it's a get next, work out what the next matching OID is.
+  case "$oid_req" in
+    $oid_prefix | $oid_prefix.0 | $oid_prefix.0.* | $oid_prefix.1)
+      oid_req=$oid_prefix.1.0;;
+
+    $oid_prefix.1.* | $oid_prefix.2 | $oid_prefix.2.0)
+      oid_req=$oid_prefix.2.0;;
+
+    $oid_prefix.2.* | $oid_prefix.3 | $oid_prefix.3.0)
+      oid_req=$oid_prefix.3.0;;
+
+    $)
+      # No more entries - just return without text and it will be handled by snmpd.
+      exit 0;;
+  esac
+elif [ "$1" = "-g" ]; then
+  # If it's an exact get, ensure that it matches one of our valid OIDs.
+  case "$oid_req" in
+    $oid_prefix.1.0 | $oid_prefix.2.0 | $oid_prefix.3.0)
+      # No matching entry - just return without text and it will be handled by snmpd.
```

Comment:
This comment reads as if it's for the line above, which i now see it isn't. Worth adding a comment for the first case stating that we handle the return stuff below, and if not these three oids, we return.